components/Extensions/src/Text/StringExtensions.Localization.cs

```diff
@@ -49,16 +49,16 @@ static StringExtensions()
     /// You can retrieve this from a UIElement.UIContext, XamlRoot.UIContext (XamlIslands), or Window.UIContext.</param>
     /// <returns>string value for given resource or empty string if not found.</returns>
     ////[SupportedOSPlatform("Windows10.0.18362.0")]
-    public static string GetViewLocalized(this string resourceKey, UIContext? uiContext = null)
+    public static string? GetViewLocalized(this string resourceKey, UIContext? uiContext = null)
     {
         if (uiContext != null)
         {
             var resourceLoader = ResourceLoader.GetForUIContext(uiContext);
-            return resourceLoader.GetString(resourceKey);
+            return resourceLoader?.GetString(resourceKey) ?? string.Empty;
```

Comment:
There shouldn't be a null check here. Also the method already returns nullable, we shouldn't do `?? string.Empty`
```suggestion
            return resourceLoader.GetString(resourceKey);
```
Same for the other branch below.